### PR TITLE
Add JVM argument support

### PR DESCRIPTION
**Pull request by jqhan**

### CODE REVIEW

**Comment by KengoTODA** [comment]
Thank you. Please update CHANGELOG.md too.

**Comment by jqhan** [comment]
Alright. Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ Currently the versioning policy of this project follows [Semantic Versioning](ht
 
 ## Unreleased - 2018-??-??
 
+## 1.7.0 - 2018-07-11
+
+* Use SpotBugs 3.1.3
```
Comment (KengoTODA) [inline CHANGELOG.md]: please remove this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ Currently the versioning policy of this project follows [Semantic Versioning](ht
 
 ## Unreleased - 2018-??-??
 
+## 1.7.0 - 2018-07-11
```
Comment (KengoTODA) [inline CHANGELOG.md]: this line is also needless, we add it when release new version.

**Comment by GunnarI** [comment]
When will this fix be merged?

**Comment by davidburstromspotify** [comment]
@KengoTODA Could you please merge the changes (and subsequently release them), our transition to SpotBugs is blocked on this. Once we've moved to SpotBugs, it will likely lead to more contributed improvements on the plugin.

**Comment by KengoTODA** [comment]
sorry for my laziness, I've merged.

**Pull request merged**